lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -185,6 +185,7 @@ def create_and_submit(self):
         user = UserFactory.create()
         attempt = SoftwareSecurePhotoVerification(user=user)
         user.profile.name = u"Rust\u01B4"
+        user.username = u"Rust"
```

Comment:
As this username will be part of exception message, so I need to have username defined.